client/src/RestaurantList/RestaurantList.jsx

```diff
@@ -0,0 +1,35 @@
+// @flow
+import React, { Component } from 'react';
+import { RestaurantCard } from './RestaurantCard';
+import type { Restaurant } from '../flowTypes';
+
+type State = Array<Restaurant>
+
+class RestaurantList extends Component<{}, State> {
+  state = {
+    restaurants: [{
```

Comment:
read about faker.js :)

Reply:
Cool! Doesn't contain restaurant/cuisines though. The ruby one does, I'll use it on the server side